src/mscorlib/src/System/ArgIterator.cs

```diff
@@ -133,5 +133,54 @@ public override bool Equals(Object o)
             
         private IntPtr ArgPtr;                  // Pointer to remaining args.
         private int    RemainingArgs;           // # of remaining args.
+#else
+        public ArgIterator(RuntimeArgumentHandle arglist)
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator see: https://github.com/dotnet/coreclr/issues/9204.
```

Comment:
I asked them to do it this way as periodically we wantto grep for PNSE and this way it appears conveniently in the output, I dont have to go visit the code.